wiki/en/en-Walk-Through.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: wiki
+title: "Jamulus Walk-Through"
+lang: "en"
+permalink: "/wiki/Walk-Through"
+---
+# Jamulus walk-through
+Let's have a look at what Jamulus looks like and what you should do after having installed the software.
+## First start
+Before you start Jamulus:
+1. Close all other applications
+2. Plugin everything (Ethernet cable, audio gear, etc.)
+
+... to minimise CPU load and to ensure that no other application tries to use your sound card.
+
+You can have a look at [Hardware Setup](Hardware-Setup) for more information on how to set up your hardware. If you need help setting up ASIO4All on Windows, have a look at the [ASIO4All Setup guide on the Windows installation page](Installation-for-Windows#setting-up-asio4all).
```

Comment:
Stylistically a bit awkward to repeat "set up hardware" in this way, so I'd say "You can have a look at [Hardware Setup](Hardware-Setup) for more information on what to do with that."

Reply:
Changed. But now it's a bit less clear in my opinion (since the reader must think more)

Reply:
OK how about "You can have a look at Hardware Setup for more information on what to do with your equipment." ?

Reply:
Yes. Changed that.